lec16-data-viz.Rmd

```diff
@@ -1,4 +1,97 @@
 ---
-title: "More on data visualizations (plus project work)"
+title: "More on data visualizations and the git workflow (plus project work)"
```

Comment:
Git

Reply:
I actually thought it was always lower case, as in their logo, but apparently it is always with a capital 'G'. Why would they make a lower case logo and then go with that convention... (>_<)

Reply:
Haha, because the lower case logo is "edgy" and "cool" :P